MAINTAINERS.yml

```diff
@@ -4556,6 +4556,19 @@ Panasonic Platforms:
   labels:
     - "platform: Panasonic"
 
+"Python Tooling: ruff":
+  status: maintained
+  maintainers:
+    - pdgendt
+  collaborators:
+    - kartben
+  files:
+    - .ruff.toml
+    - .ruff-excludes.toml
+    - scripts/ruff/
+  labels:
+    - "area: Python Tooling"
```

Comment:
Do we really need a dedicated area with a "maintainer" for just some linter configurations?

IMHO, this is one of those things that should be considered "sub-area" -- given that we do not currently support the "sub-area" scheme in the MAINTAINERS file, this should just be part of the `Coding Guidelines` area.